src/javascript/app_2/App/Components/Layout/Footer/toggle_fullscreen.jsx

```diff
@@ -0,0 +1,61 @@
+import React   from 'react';
+import Popover from '../../../Components/Elements/popover.jsx';
+
+class ToggleFullScreen extends React.Component {
+    constructor(props) {
+        super(props);
+        this.toggleFullScreen = this.toggleFullScreen.bind(this);
```

Comment:
@zakibinary 
it doesn't need to bind the `toggleFullScreen` again in constructor since it's defined as an arrow function.

Reply:
`onFullScreen` is defined as an arrow function, not `toggleFullScreen`.